scripts/generate-documents/templates/validation.ts

```diff
@@ -0,0 +1,49 @@
+import {type DocGenTemplate} from '../types'
+import {lorem} from '../utils/lorem-bytes'
+
+export const validation: DocGenTemplate = (options) => ({
+  _type: 'validationTest',
+  arrayOfSlugs: [
+    {
+      _type: 'slugEmbed',
+      sku: {
+        _type: 'slug',
+        current: 'foo',
+      },
+    },
+  ],
+  body: [
+    {
+      _type: 'block',
+      children: [
+        {
+          _type: 'span',
+          marks: [],
+          text: lorem(options.size / 2),
```

Comment:
going to be honest the names of these functions felt a bit cryptic initially 😄 
I think we could benefit from having a small comment / a slight renaming over on [lorem-bytes.ts](https://github.com/sanity-io/sanity/pull/7387/files#diff-ea11848b0d77a555763f12acfdcda8868bf96c4b20b653ff72b2fdc8d797ca98) ?

Reply:
what was cryptic about them? what would be a better name?

Reply:
I think that in terms of naming it could be something related more to `getSampleText` or something along those lines (though I'm not married to the name, the name `lorem` does invoke sample / placeholder text)

However, my first gut feeling was: why do I have to pass the `options.size` into this, and what does it have to do with the sampling. Just to confirm that I understood it, this is more about making sure that the content in this PTE is not all the same text so that there is some variation between the docs generated, right?

Reply:
Renamed `lorem` to `loremString` and added some tsdocs here – hopefully that made things a bit clearer: https://github.com/sanity-io/sanity/commit/c0b4a8031f59537d6a210c7696cb789b0245b79d

Reply:
getSampleText would be ok, but loremText feels more specific.

> Just to confirm that I understood it, this is more about making sure that the content in this PTE is not all the same text so that there is some variation between the docs generated, right?

Yep! Sometimes you might want to generate large documents, so this provides the ability to specify the desired byte size of the documents you're generating.